aspnetcore/fundamentals/hosting.md

```diff
@@ -250,33 +559,360 @@ using (host)
 }
 ```
 
-The URL formats that are valid here depend on the server you're using. For more information, see [Server URLs](#server-urls) earlier in this article.
+You can initialize and start a new host using the pre-configured defaults of `CreateDefaultBuilder` using a static convenience method. These methods start the server without console output and with [WaitForShutdown](/dotnet/api/microsoft.aspnetcore.hosting.webhostextensions.waitforshutdown) wait for a break (Ctrl-C/SIGINT or SIGTERM):
 
-> [!NOTE]
-> The `UseConfiguration` extension method isn't currently capable of parsing a configuration section returned by `GetSection` (for example, `.UseConfiguration(Configuration.GetSection("section"))`. The `GetSection` method filters the configuration keys to the section requested but leaves the section name on the keys (for example, `section:urls`, `section:environment`). The `UseConfiguration` method expects the keys to match the `WebHostBuilder` keys (for example, `urls`, `environment`). The presence of the section name on the keys prevents the section's values from configuring the host. This issue will be addressed in an upcoming release. For more information and workarounds, see [Passing configuration section into WebHostBuilder.UseConfiguration uses full keys](https://github.com/aspnet/Hosting/issues/839).
+**Start(RequestDelegate app)**
+
+Start with a `RequestDelegate`:
+
+```csharp
+using (var host = WebHost.Start(app => app.Response.WriteAsync("Hello, World!")))
+{
+    host.WaitForShutdown();
+
+    Console.WriteLine("Press any key to exit...");
+    Console.ReadKey();
+}
+```
 
-### Ordering Importance
+Make a request in the browser to `http://localhost:5000` to receive the response "Hello World!" Issuing a break (Ctrl-C/SIGINT or SIGTERM) in the console window bypasses the `WaitForShutdown`. The app displays the `Console.WriteLine` message and waits for a keypress to exit.
```

Comment:
nit: missing period after `Hello World`? Also, bypass sounds a little awkward here how about: "`WaitForShutdown` blocks until a break (Ctrl-C/SIGINT or SIGTERM) is issued." or some variant of that?

Reply:
> missing period after Hello World?

The exclamation point is the terminal punctuation mark that ends the sentence.

> `WaitForShutdown` blocks until a break (Ctrl-C/SIGINT or SIGTERM) is issued.

:+1: